### DISCUSSION TITLE
Branding is coming...

### DISCUSSION THREAD

**Discussion by bitbound** [immense/Remotely#232] [Show and tell]
Plus compile-free retrieval of server URL and organization IDs in the installer and quick support EXEs.

These are made possible by an web API service that I'm now hosting and will be available to GitHub sponsors.

![image](https://user-images.githubusercontent.com/20995508/107161010-95693e00-694e-11eb-93e4-83aca99bd964.png)

![image](https://user-images.githubusercontent.com/20995508/107161014-9ac68880-694e-11eb-8b86-9697ced58811.png)

![image](https://user-images.githubusercontent.com/20995508/107161016-a1ed9680-694e-11eb-971a-e33372f66d20.png)

**Comment by bitbound** [comment]
Also, server upgrade notifications and new upgrade scripts.

![image](https://user-images.githubusercontent.com/20995508/107161076-0dcfff00-694f-11eb-8fc0-861f23b12dc4.png)

**Comment by demirozmustafa** [comment]
Multi tenancy ?

**Reply by bitbound** [reply]
Yes.

**Comment by bitbound** [comment]
After further consideration, I decided to not host a separate API for providing the relay.  Instead, relay and branding will work using your own self-hosted server, but only if you compiled the server URL into the clients.

**Comment by RDFonseca82** [comment]
this is now available?

**Reply by bitbound** [reply]
> this is now available?

Yes, it is.  However, it requires the project to be built from source, with the hostname embedded.
